homework/01LessonOne.js

```diff
@@ -202,12 +202,12 @@ var stringFourStrictNotEqualsNumberFour;
 //
 // What’s the difference between == and ===?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here:== isn't as strict. it will try to convet both sides to the same type. === is strict, and will not convert anything before comparing.
 //
 //
 // What’s the difference between != and !==?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: Same concept as above.  != (not equal to) is less strict than !==. For example, if x=2, x != '2' would be true, while x !== '2' is false. Right?
 //
```

Comment:
Correct! Simply put, "!=" checks for a difference in value only. "!==" checks for a difference in value AND/OR data type.